core/templates/services/promo-bar-backend-api.service.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service Promo bar.
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ServicesConstants } from 'services/services.constants';
+
+export interface PromoBarData {
```

Comment:
Make it PromoBar (or something around that, let's avoid Data as it doesn't add any info to the class name.)

Optional: I would suggest creating a PromoBar class.

Reply:
Done!